requirements/constraints.txt

```diff
@@ -14,9 +14,6 @@
 # urllib3>=2.0.0 conflicts with zipp so we're using the same version as in edx-platform
 urllib3<2.0.0
```

Comment:
Have you checked if edx-platform still uses this version?

Reply:
Apparently this is still used, so I think we can resolve this conversation
https://github.com/openedx/edx-platform/blob/30039845a403b8d1c323440a24907c2677f1be1e/scripts/user_retirement/requirements/base.in#L17